apps/commercetools/LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License
```

Comment:
Why removed?

Reply:
When I replace the old dir for the new one it was in the diff, added again